tests/test_equi_gen_force_field.py

```
@@ -40,6 +40,24 @@ def test_deepmd_append(self):
         ret2 = dpti.equi.gen_equi_force_field(**input)
         self.assertEqual(ret1, ret2)
 
+    def test_template_ff(self):
+        input = {
```

Ruff (0.15.21)

[error] Variable `input` is shadowing a Python builtin

(A001)